src/pages/index.tsx

```diff
@@ -1,5 +1,291 @@
-import { Lp2021 } from '../components/page/Lp2021'
+import React, { useCallback } from 'react'
+import type { NextPage, GetStaticProps } from 'next'
+import Head from 'next/head'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
 
-const IndexPage: React.FC = () => <Lp2021 />
+import { dkClient } from '../dreamkast/client'
+import { Conference } from '../dreamkast/types'
+
+type Props = {
+  archivedConferences: Conference[]
+  registeredConferences: Conference[]
+}
+
+export const getStaticProps: GetStaticProps<Props> = async () => {
+  const conferences = await dkClient.List()
+  return {
+    props: {
+      archivedConferences: conferences
+        .filter(
+          (c: Conference) => c.status === 'archived' || c.status == 'closed',
```

Comment:
nits: strict equalに統一したいです

```suggestion
          (c: Conference) => c.status === 'archived' || c.status === 'closed',
```